StudentManagementApp.java

```diff
@@ -0,0 +1,9 @@
+public class StudentManagementApp {

```

Comment:
You can rename the java class from "StudentManagementApplication" to "Main", so we can easier find the entry / starting point for our application.